addons/sys_signal/fnc_getRadioClimateCode.sqf

```diff
@@ -0,0 +1,36 @@
+#include "script_component.hpp"
+/*
+ * Author: ACRE2Team
+ * Returns the Longley-Rice (ITM) or ITWOM radio climate code for a specific map.
+ *
+ * Arguments:
+ * 0: Map <STRING>
+ *
+ * Return Value:
+ * Radio climate code <NUMBER>
+ *
+ * Example:
+ * ["Altis"] call acre_sys_signal_fnc_getRadioClimateCode
+ *
+ * Public: No
+ */
+
+params [["_map", ""]];
+
+if (_map == "") exitWith {MAP_CLIMATE_CONTINENTAL_TEMPERATE};
+
+_map = toLower _map;
+
+private _config = configProperties [configFile >> "CfgAcreSignal" >> "LongleyRiceRadioClimate", "isArray _x", true];
```

Comment:
Do we want to support this config in the map mod itself?

Reply:
Better not...